Xamarin.Forms.Platform.UAP/ImageSourceConverter.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using Windows.UI.Xaml.Controls;
+
+namespace Xamarin.Forms.Platform.UWP
+{
+	internal class ImageSourceConverter : Windows.UI.Xaml.Data.IValueConverter
+	{
+		public object Convert(object value, Type targetType, object parameter, string language)
+		{
+			// TODO: investigate whether we can use AsyncValue<T> instead of blocking
```

Comment:
We should either do this now or open up a separate GitHub issue to track this; otherwise, this TODO will sit here forever.

Reply:
To be completely honest I just copy/pasted it from the other value converter that was previously being used and didn't even notice it. I'll see if I can fix it or create an issue for the both of them

Reply:
Fixed for the "new" icon. For the "old" one I don't think it's possible, so removed the TODO.